ods_ci/tests/Tests/0500__ide/0502__ide_elyra.robot

```diff
@@ -55,10 +55,9 @@ Verify Pipelines Integration With Elyra When Using Standard Data Science Based I
     [Template]    Verify Pipelines Integration With Elyra Running Hello World Pipeline Test
     [Tags]        Tier1    ODS-2271
     [Timeout]     30m
```

Comment:
is this timeout applied to each single test using the template? O it is the overall timeout of the full test?

Reply:
@bdattoma I also wasn't sure as these aren't exactly clear:
* https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#test-case-timeout
* https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#test-templates

So, I tried that and it turned out that this is an overall timeout for the full test. So all these template cases need to be finished within this timeout together.